src/services/jwt.service.js

```diff
@@ -0,0 +1,37 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY);
+
+  return token;
+}
+
+function verify(token) {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch(e) {
+    return null;
+  }
+}
+
+function signRefresh(user) {
+  const token = jwt.sign(user, process.env.JWT_REFRESH_KEY);
```

Comment:
Ensure that the `user` object passed to `jwt.sign` for refresh tokens does not contain sensitive information such as passwords. Consider extracting only necessary fields (e.g., user ID, email) before signing the token.